docs/language/accounts.mdx

```diff
@@ -88,6 +88,19 @@ to the `prepare` phase of the transaction.
 
       let keys: AuthAccount.Keys
 
+      // Key management
```

Comment:
Is this supposed to include the public key API changes?

Reply:
This PR just reverts the removal of the deprecated functions. This documentation should have been added back to master already before, but it seems like it wasn't